odxtools/odxtypes.py

```diff
@@ -254,5 +254,4 @@ def __str__(self) -> str:
         elif self in (DataType.A_UNICODE2STRING, DataType.A_ASCIISTRING, DataType.A_UTF8STRING):
             return "string"
         else:
-            odxraise(f"Type info for type {self.value}", NotImplementedError)
-            return "<unknown type>"
+            return "<unknown type '{self.value}'>"
```

Comment:
```suggestion
            return f"<unknown type '{self.value}'>"
```

Reply:
arrg. I'm never too stupid for such mistakes. fixed, thanks!